opencsp/app/sofast/lib/process_optics_geometry.py

```diff
@@ -410,6 +410,37 @@ def process_multifacet_geometry(
     list[cdc.CalculationImageProcessingFacet],
     cdc.CalculationError,
 ]:
+    """Processes optic geometry for multifacet deflectometry measurement
+
+    Parameters
+    ----------
+    facet_data : DefinitionFacet
+        Facet definition object
+    ensemble_data : DefinitionEnsemble
+        Ensemble definition object
+    mask_raw : ndarray
+        Raw calculated mask
```

Comment:
What is the type of the mask? Is it a boolean WxH array?

Reply:
Documentation updated